spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinding.java

```diff
@@ -38,15 +38,22 @@
 
 	protected final T target;
 
-	protected final Lifecycle endpoint;
-
-	public DefaultBinding(String name, String group, T target, Lifecycle endpoint) {
+	protected final Lifecycle runningComponent;
+
+	/**
+	 * Creates a new instance.
```

Comment:
I was thinking something simple description like `Creates an instance that contains information about the binding of the producer/consumer target`